tests/test_end_to_end_dcp.py

```diff
@@ -232,6 +263,18 @@ def _run_command(cmd_and_args_list, expected_retcode=0):
 class TestEndToEndDCP(unittest.TestCase):
 
     SMARTSEQ2_BUNDLE_PATH = 'tests/fixtures/bundles/Smart-seq2'
+    EXPECTED_FILES_IN_SS2_RESULT_BUNDLE_REGEXES = [
+        re.compile('^Aligned\.sortedByCoord\.out\.bam$'),
+        re.compile('^Aligned\.toTranscriptome\.out\.bam$'),
+        re.compile('^.+_rna_metrics$'),
+        re.compile('^.+_alignment_metrics$'),
+        re.compile('^.+\.genes\.results$'),
+        re.compile('^.+\.isoforms\.results$'),
+        re.compile('^.+\.gene\.expected_counts$'),
+        re.compile('^.+\.gene\.counts\.txt$'),
+        re.compile('^.+\.exon\.counts\.txt$'),
+        re.compile('^.+\.transcripts\.counts\.txt$')
+    ]
```

Comment:
These regex work well for now, I comment here just to point out some possible future changes:
1. Even we are not validating the full filenames here, some of the filename may change significantly and the regex might expire at that time. (For example, have a look at here: https://github.com/HumanCellAtlas/skylab/blob/f1c086af250ae4808ac9f61deca8a237e6b224e5/smartseq2_single_sample/tasks/test_wdl/test_hisat2.wdl#L30)

2. We may also want to separate the regex result checklist from the code into a, for example, JSON file, so later even the output changes, we could easily change this list.